src/window.rs

```diff
@@ -767,6 +767,19 @@ impl Window {
     pub fn request_user_attention(&self, request_type: Option<UserAttentionType>) {
         self.window.request_user_attention(request_type)
     }
+
+    /// Returns the visible state of the window. This is `true` if some part of the window
+    /// is visible on screen and false otherwise. This is useful for pausing rendering
+    /// when the window isn't visible since some platforms may behave strangely if you
+    /// try to draw to an occluded window.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / Windows / X11 / Wayland:** Unsupported and always returns false
+    #[inline]
+    pub fn visible(&self) -> bool {
```

Comment:
I'm not sure if this is the best name. The only alternative I can think of is `occluded`, but that inverts the logic.

Reply:
This is intended as a way to see whether we should render to the window, so maybe name it as such: `should_render`?

Reply:
Yeah that's what it is. We could name it occluded but that makes it confusing since it could be partially occluded or something else. `should_render` could work but it feels weird to name it after the purpose of which it exists and what we think the user will do with it rather than what it actually is. Describing the purpose of it via docs as I did feels like it makes more sense but I am happy to change it. I based the naming on how the rest of the api is named after what it is rather than what it's for. @madsmtm

Reply:
I can see your reasoning, `visible` is fine to me